<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>publish_layer</class>
 <widget class="QDialog" name="publish_layer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Publish layer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_layer">
     <property name="text">
      <string>Anda akan mempublish layer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_unduh">
     <property name="text">
      <string>Aktifkan Pengunduhan untuk layer ini?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmb_download"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_tutup">
       <property name="text">
        <string>Tutup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_proses">
       <property name="text">
        <string>Proses!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_progress">
     <property name="styleSheet">
      <string notr="true">background-color:yellow;
padding:5px;
color:black</string>
     </property>
     <property name="text">
      <string>Layer sedang dipublish</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
